apis/v1alpha1/tlsroute_types.go

```diff
@@ -59,39 +58,9 @@ type TLSRouteRule struct {
 	// if **any** one of the matches is satisfied.
 	// +optional
 	Matches []TLSRouteMatch `json:"matches"`
```

Comment:
Add a max limit of 16 like we have done in HTTPRoute.

Reply:
Updated, thanks!